docs/spec/spec.md

```diff
@@ -96,7 +88,7 @@ The command will generate [Derived Entity Types and Persist Clients](https://git
 as per the `persist` specification.
 Additionally, this command will create the database schema associated with the data model definition.
 
-It will add generated files under the conventions, 
+It will add generated files under the conventions,
 1. If the file name is the same as the package name, it will generate the files under the `default` module.
```

Comment:
```suggestion
1. If the model name is the same as the package name, it will generate the files under the `default` module.
```

Reply:
updated in [c6f8aa2](https://github.com/ballerina-platform/persist-tools/pull/148/commits/c6f8aa200f6cd6b2665e3f091a75d12c1ef59a0f)